### DISCUSSION TITLE
MCP2515 for raspberry pi pico

### DISCUSSION THREAD

**Discussion by IsaNocNoc** [micropython/micropython#14503] [Libraries & Drivers]
Hi, I am trying to connect 2 raspberry pi pico with mcp2515 CAN module but I can't find any working library or documentation for micropython. Can anyone provide me with a good MCP2515 library and how I could program it in micropython?

**Comment by scruss** [comment]
There are two listed in [Awesome MicroPython](https://awesome-micropython.com/#can)
